src/routes/docs/tutorials/nextjs-ssr-auth/step-3/+page.markdoc

```diff
@@ -20,7 +20,7 @@ export async function createSessionClient() {
     .setEndpoint(process.env.NEXT_PUBLIC_APPWRITE_ENDPOINT)
     .setProject(process.env.NEXT_PUBLIC_APPWRITE_PROJECT);
 
-  const session = cookies().get("my-custom-session");
+  const session = await cookies().get("my-custom-session");
```

Comment:
Should it be `(await cookies()).get("my-custom-session");` ?

Reply:
Isn't it the same? The `await` is for `cookies()`. Wrapping it with the parentheses makes it more clear though.
Maybe using `await cookies()?.get("my-custom-session")` or even wrap into a `trycatch`?